wormhole-connect/src/config/types.ts

```diff
@@ -74,8 +74,36 @@ export interface WormholeConnectConfig {
   bridgeDefaults?: BridgeDefaults;
   routes?: string[];
   pageHeader?: string;
+  pageSubHeader?: string;
+  searchTx?: SearchTxConfig;
 }
 
+export type SearchTxConfig = {
+  txHash?: string;
+  chainName?: ChainName;
+};
+export type MoreTokenConfig = {
+  label: string;
+  href: string;
+};
+
+export type ExtraChainConfig = {
```

Comment:
Should this extra chain/token types be on this PR?

Reply:
oh, good call, thanks